official/resnet/resnet_run_loop.py

```diff
@@ -53,22 +59,27 @@ def process_record_dataset(dataset, is_training, batch_size, shuffle_buffer,
     parse_record_fn: A function that takes a raw record and returns the
       corresponding (image, label) pair.
     num_epochs: The number of epochs to repeat the dataset.
-    num_parallel_calls: The number of records that are processed in parallel.
-      This can be optimized per data set but for generally homogeneous data
-      sets, should be approximately the number of available CPU cores.
     examples_per_epoch: The number of examples in the current set that
       are processed each epoch. Note that this is only used for multi-GPU mode,
       and only to handle what will eventually be handled inside of Estimator.
-    multi_gpu: Whether this is run multi-GPU. Note that this is only required
-      currently to handle the batch leftovers (see below), and can be removed
-      when that is handled directly by Estimator.
+    use_distribution_strategy: Whether DistributionStrategies API is used.
+    gpus_for_distribution_strategy: How many GPUs are used with
+      DistributionStrategies.
 
   Returns:
     Dataset of (image, label) pairs ready for iteration.
   """
+
+  # TODO(taylorrobie@) remove when DistributionStrategies uses global batch size
```

Comment:
Let's not leave TODOs in the public code; a comment explaining that this is only necessary for a short period because etc etc is sufficient. Also, nit, for future reference: TODO(taylorrobie), and, this is public code, so, robieta.